proto/agent/agent.proto

```diff
@@ -14,87 +14,11 @@
 
 syntax = "proto3";
 
-// Retransmit?
-// Sliding windows?
+option go_package = "sigs.k8s.io/apiserver-network-proxy/proto/agent";
 
-
-service ProxyService {
-  rpc Proxy(stream Packet) returns (stream Packet) {}
-}
+import "konnectivity-client/proto/client/client.proto";
```

Comment:
Eventually I think we are going to want to break out the agent.proto in a similar way. We should consider making a .proto which is the actual protocol and two service level files which each import the the protocol.